dist/tools/compile_and_test_for_board/README.md

```diff
@@ -20,6 +20,20 @@ They can be checked with:
     find results/ -name '*.failed'
     find results/ -name 'test.success'
 
+Script checks
+-------------
+
+Use [tox](https://tox.readthedocs.io/en/latest/) to run basic checks on the
+script:
+
+    $ tox compile_and_test_for_board.py
```

Comment:
Why does it have the script as argument ?
Every script could need its own `tox` file as the script may have its dependencies anyway.

Reply:
> Why does it have the script as argument ?

I find this more generic this way. We could reuse this tox config file for any python script.